src/nplinker/metabolomics/gnps/gnps_molecular_family_loader.py

```diff
@@ -19,6 +18,13 @@ def __init__(self, file: str | PathLike):
         3. FEATURE-BASED-MOLECULAR-NETWORKING
             - networkedges_selfloop/*.selfloop
 
+        The "ComponentIndex" column in the GNPS molecular family file is treated
+        as family id. But for molecular families that have only one member (i.e. spectrum),
+        which are called singleton molecular family, they have the same value of
```

Comment:
About "they have the same value of", please specify which field you're referring to (I think family id)